src/main/java/com/internet/shop/controllers/products/AdminProductController.java

```diff
@@ -0,0 +1,27 @@
+package com.internet.shop.controllers.products;
+
+import com.internet.shop.lib.Injector;
+import com.internet.shop.model.Product;
+import com.internet.shop.service.ProductService;
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class AdminProductController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("com.internet.shop");
+    private final ProductService productService =
+            (ProductService) injector.getInstance(ProductService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
```

Comment:
after adding a new product to the db, we should redirect to this controller

Reply:
Done